index.html

```diff
@@ -30,6 +29,176 @@ <h1 class="name">Javed Shahbaz</h1>
+
+        </article>
+        <article class="Cards">
+
+            <img class="imgp" src="images/project5.png" alt="P1" width="100%">
+            
+            <div class="P-Data">
+                
+                <h3>Project name goes here</h3>
+
+                <ul class="PUl">
+                    <li class="Pli">HTML/CSS</li>
+                    <li class="Pli">Ruby on Rails</li>
+                    <li class="Pli">Java script</li>
+                </ul>
+
+                <button class="move">
+                    <a class="textBoton" href="#">See this project &nbsp <img src="./images/Union.png" alt=""> </a>
+                </button>
+
+            </div>
+
+        </article>
+        <article class="Cards">
+
+            <img class="imgp" src="images/project6.png" alt="P1" width="100%">
+            
+            <div class="P-Data">
+                
+                <h3>Project name goes here</h3>
+
+                <ul class="PUl">
+                    <li class="Pli">HTML/CSS</li>
+                    <li class="Pli">Ruby on Rails</li>
+                    <li class="Pli">Java script</li>
+                </ul>
+
+                <button class="move">
+                    <a class="textBoton" href="#">See this project &nbsp <img src="./images/Union.png" alt=""> </a>
+                </button>
+
+            </div>
+
+        </article>
+    </div>
+    </section>
+
+
+
+    <section class="About">
+
+        <h2>About</h2>
+        
+        <p class="About">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            
+        <button class="blackBoton">
+                <a class="textBoton" href="#"> Get my CV &nbsp</a>
+                <img class="get-my-resume-image" src="./images/ic_download.svg">
+        </button>
+        
+        <section class="boxL">
+            <ul class="Box languages">
+                <h4>Languages</h4><br>
+                <li class="inBox">JavaScript(ES5/ES6)</li>
+                <li class="inBox">Ruby</li>
+                <li class="inBox">HTML</li>
+                <li class="inBox">CSS</li>
+            </ul>
+            <ul class="Box framework">
+                <h4>Frameworks</h4><br>
+                <li class="inBox">Bootstrap</li>
+                <li class="inBox">Ruby on rails</li>
+                <li class="inBox">RSpec</li>
+                <li class="inBox">Capibara</li>
+                <li class="inBox">Selenium</li>
+            </ul>
+            <ul class="Box skills">
+                <h4>Skills</h4><br> 
+                <li class="inBox">Codekit</li>
+                <li class="inBox">Codepen</li>
+                <li class="inBox">GitHub</li>
+                <li class="inBox">CitLAb</li>
+                <li class="inBox">Terminal</li>
+            </ul>
+        </section> 
```

Comment:
- [ ] Well done 👍 on implementing the changes the previous reviewer asked for. 
  - One issue is still not resolved. The reviewer suggested you follow the design guideline for margin and spacing also. In the [given design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=gKzI1MhWX8MvOOH4-0), `Get my CV` button, `Languages`, `Frameworks` and `Skills` card should be in the center of the screen and should have the same margin/padding on the `left and right` side.
  - ***Left picture is your design, and the right one is the given design.***
  ![image](https://user-images.githubusercontent.com/60083437/219811602-cd1cee5e-335d-4184-8d2d-45f7b651c2c0.png)